connectors/migrations/20240717_migrate_github_installation_id.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE github_connector_states
+SET "installationId" = (
+    SELECT "connectionId"
```

Comment:
had no idea you could subquery in an update query

Reply:
yet to be tested :) risk of failure is low even in prod since it's a new unused column

Reply:
will test it in dev before of course

Reply:
Tested in dev looks like it worked